File: DistanciaPuntos/DistanciaPuntosCiSharp/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace DistanciaPuntosCiSharp
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void btnCalcular_Click(object sender, EventArgs e)
        {
            Punto1 P1 = new Punto1(Double.Parse(tbX1.Text), Double.Parse(tbY1.Text));

            tbResultado.Text = P1.DistanciaPunto2(Double.Parse(tbX2.Text), Double.Parse(tbY2.Text)).ToString();
        }
    }
}

File: DistanciaPuntos/DistanciaPuntos/Form1.vb
Public Class Form1
    Private Sub btnCalcular_Click(sender As Object, e As EventArgs) Handles btnCalcular.Click
        Dim P1 As New Punto3D(tbX1.Text, tbY1.Text, tbZ1.Text)

        tbResultado.Text = P1.CalcularDistancia(New Punto3D(tbX2.Text, tbY2.Text, tbZ2.Text))
    End Sub

End Class
